lib/realtime_web/channels/realtime_channel.ex

```diff
@@ -54,6 +55,11 @@ defmodule RealtimeWeb.RealtimeChannel do
       |> assign(:private?, !!params["config"]["private"])
       |> assign(:policies, nil)
 
+    case Join.validate(params) do
+      {:ok, _join} -> nil
+      {:error, :invalid_join_payload, errors} -> log_error(socket, "InvalidJoinPayload", errors)
```

Comment:
Should it also log the params? Otherwise we will see errors but we won't see the original params that caused the errors I think?

Reply:
logging them out with redacted values for user_token and access_token